### PR TITLE
Update _list-row-empty-state.html include to use getPreferredVersion for canI check

### PR DESCRIPTION
**Pull request by benjaminapetersen**
....Almost done with the includes.

picks off one in `/overview/_list-row-content.html` as well.

### CODE REVIEW

File: app/views/overview/_list-row-content.html [hunk 1]
```diff
@@ -43,7 +43,7 @@ <h3>
       </span>
     </span>
     <a ng-href="project/{{row.apiObject.metadata.namespace}}/browse/events">View Events</a>
-    <span ng-if="'replicationcontrollers' | canI : 'update'">
+    <span ng-if="replicationControllersVersion | canI : 'update'">
```
Comment (spadgett) [inline app/views/overview/_list-row-content.html]: row.replicationControllersVersion

File: app/views/overview/_list-row-empty-state.html [hunk 2]
```diff
@@ -28,7 +28,7 @@ <h2>No deployments.</h2>
       </div>
     </div>
     <div ng-if="!row.pipelines.length">
-      <button ng-if="'deploymentconfigs/instantiate' | canI : 'create'"
+      <button ng-if="deploymentConfigsInstantiateVersion | canI : 'create'"
```
Comment (spadgett) [inline app/views/overview/_list-row-empty-state.html]: row.deploymentConfigsInstantiateVersion

**Comment by benjaminapetersen** [comment]
@spadgett bah, sorry about that. I went too late last night these last few.  I'll run through them again.

**Comment by benjaminapetersen** [comment]
updated

**Pull request merged**